packages/bpk-component-breakpoint/readme.md

```diff
@@ -14,10 +14,34 @@ npm install bpk-component-breakpoint --save-dev
 import React from 'react';
 import BpkBreakpoint, { BREAKPOINTS } from 'bpk-component-breakpoint';
 
+const activeStyle = {
+  padding: spacingBase,
+  backgroundColor: colorGreen500,
+};
+
+const inactiveStyle = {
+  padding: spacingBase,
+  backgroundColor: colorGray100,
+};
+
 export default () => (
-  <BpkBreakpoint query={BREAKPOINTS.MOBILE}>
-    {isActive => (isActive ? <span>Mobile viewport is active</span> : <span>Mobile viewport is inactive</span>)}
-  </BpkBreakpoint>
+  <div>
+    <BpkBreakpoint query={BREAKPOINTS.MOBILE}>
+      {isActive => <div style={isActive ? activeStyle : inactiveStyle}>{'MOBILE'}</div>}
```

Comment:
This is an interesting one, because I'd immediately looked at that code and thought that `<div style={isActive ? activeStyle : inactiveStyle}>` should be encapsulated in a component i.e.

```
const BreakpointIndicator = (props) => (
  <BpkBreakpoint query={props.query}>
    <div style={props.isActive ?{...} : {...}}>{props.children}</div>
  </BpkBreakpoint>
);
```

However...

It's perhaps a little too abstract for the purposes of demonstrating the usage of `BpkBreakpoint`. Perhaps 

```
const BreakpointIndicator = (props) => (
    <div style={props.isActive ? {...} : {...}}>{props.children}</div>
);
```